Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -369,7 +390,7 @@ - (void)prepareForReuse
     self.thread = nil;
     self.contactsManager = nil;
 
-    [self.unreadBadge removeFromSuperview];
```

Comment:
~~Don't we still need to remove the unread badge~~

Never mind, this is correct, but I think calling this the `unreadBadgeContainer` instead of `unreadPlaceholder` better communicates what's going on here.

Reply:
Done.